tests/benchmarks/test_benchmark_vpg.py

```diff
@@ -0,0 +1,292 @@
+'''
+This script creates a regression test over
+garage-PyTorch-VPG, garage-TF-VPG and baselines-VPG.
+It get Mujoco1M benchmarks from baselines benchmark, and test each task in
+its trial times on garage model and baselines model. For each task, there will
+be `trial` times with different random seeds. For each trial, there will be two
+log directories corresponding to baselines and garage. And there will be a plot
+plotting the average return curve from baselines and garage.
+'''
+
+import datetime
+import json
+import os.path as osp
+import random
+
+from baselines.bench import benchmarks
+import dowel
+import dowel.logger as dowel_logger
+import gym
+import matplotlib.pyplot as plt
+import pandas as pd
+import pytest
+import tensorflow as tf
+import torch
+
+from garage.envs import normalize
+from garage.experiment import deterministic, LocalRunner
+from garage.np.baselines import LinearFeatureBaseline
+from garage.tf.algos import VPG as TF_VPG
+from garage.tf.envs import TfEnv
+from garage.tf.experiment import LocalTFRunner
+from garage.tf.policies import GaussianMLPPolicy as TF_GMP
+from garage.torch.algos import VPG as PyTorch_VPG
+from garage.torch.policies import GaussianMLPPolicy as PyTorch_GMP
+from tests import helpers as Rh
+from tests.fixtures import snapshot_config
+
+hyper_parameters = {
+    'hidden_sizes': [64, 64],
+    'center_adv': True,
+    'learning_rate': 1e-2,
+    'discount': 0.99,
+    'n_epochs': 250,
+    'max_path_length': 500
+}
+
+
+def run_garage_pytorch(env, seed, log_dir):
+    '''
```

Comment:
while you're here, please update these comments to conform to CONTRIBUTING.md

Reply:
change the comments